near-sdk-macros/src/core_impl/code_generator/item_enum_info.rs

```diff
@@ -0,0 +1,14 @@
+use syn::ItemEnum;
+
+pub fn generate_proxy_enum(input: &ItemEnum) -> proc_macro2::TokenStream {
+    use quote::{format_ident, quote};
+    let ident = &input.ident;
+    let new_name = format_ident!("{}Contract", ident);
+    let name = quote! {#new_name};
```

Comment:
You don't need to quote the ident, it is interpolatable by itself

```suggestion
    let name = format_ident!("{}Contract", ident);
```